doc/auth.md

```diff
@@ -0,0 +1,71 @@
+# Enable Authentication
+
+Client can communicate with Pravega in a more secure way by enabling Authentication.
+Authentication Handler can be implemented in different ways. For details see [Pravega AUthentication](https://github.com/pravega/pravega/blob/master/documentation/src/docs/auth/auth-plugin.md) for details.
+
+PasswordAuthHandler(default) is the most basic authentication handler.
+For using this, user needs to do the following:
+
+1. Create a userPassword file containing `<user>:<password>:<acl>;` on each line for one user.
+Delimiter should be `:` with `;` at the end of each line.
+Use the   [PasswordCreatorTool](https://github.com/pravega/pravega/blob/master/controller/src/test/java/io/pravega/controller/auth/PasswordFileCreatorTool.java) to create a new file with the password encrypted. Use this file when creating the secret in next step.
```

Comment:
Should this point to the documentation at pravega.io site instead of Github master? I guess if the two were in the same repo, a relative URL'd be used.

Reply:
I could not find any other wiki/doc explaining about the PasswordFileCreatorTool, other than this code link. In case you know of one, please add here.